src/__tests__/reducers/tagsReducer.test.js

```diff
@@ -0,0 +1,98 @@
+import tagsReducer from '../../redux/reducers/tagsReducer'
+import { ADD_TAG, EDIT_TAG, DELETE_TAG, } from '../../constants'
+
+const sortedTagList = [
+  '112',
+  'firstTag',
+  'fruits',
+  'myTagList',
+  'peoples',
+  'secondTag',
+  'sfjk',
+  'tTgs',
+  'TTTshdf',
+  'verbs'
+]
+
+describe('tagsReducer', () => {
+  test('should return a correct state after receiving a nonexistent action', () => {
+    const action = { type: 'NON_EXISTENT_ACTION', }
+    const initialState = {
+      tagsList: sortedTagList,
+    }
+
+    expect(tagsReducer(undefined, action)).toStrictEqual(initialState)
+  })
+
+  test('should add a new tag name in tagList after receiving the ADD_TAG action', () => {
```

Comment:
_'should add a new tag name in tagList'_ is enough

Reply:
This is to I can understand with which action the test has crashed